woocommerce/woocommerce-ios
# Order Filters: allow multiple status selection

Part of #5243 

## Description
Now we allow merchants to filter orders not only by single order status but also by multiple statuses, so now the endpoints will take a list of statuses instead of only one. I updated our unit tests and I also implemented a new view controller `OrderStatusFilterViewController` which will be present the order statuses and allow you to select multiple statuses at the same time or just the status `Any`.

## Testing
1. Tap the orders tab.
2. Tap the filters button.
3. You should be able to filter orders selecting multiple statuses, or just the status `Any`.

## Screenshot
<img src="https://user-images.githubusercontent.com/495617/142253295-a828aa1c-034b-4b7a-8185-53fbfe45c9e3.png" width=300 />


Update release notes:
